edflow/edflow

```diff
@@ -85,6 +85,10 @@ def main(opt, additional_kwargs):
     # Training
     if opt.train:
         if opt.checkpoint is not None:
+            if not os.path.exists(opt.checkpoint):
```

Comment:
good idea but we cannot really do this: for tensorflow the specified checkpoint path is not a file

Reply:
This is also an older commit which should not have been there. I'll create a new branch/fork of master and edit those files in there.